src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStreamPacketType.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace Microsoft.Data.SqlClientX.IO
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClientX.IO
 {
     internal enum TdsStreamPacketType
     {
+        None = 0,
```

Comment:
Instead of None, what about making TdsStreamPacketType in the TdsWriteStream a nullable?

Reply:
Done